.travis.yml

```diff
@@ -7,11 +7,15 @@ sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bower's cache
 
 env:
+  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
   - EMBER_TRY_SCENARIO=default
```

Comment:
@sir-dunxalot `EMBER_TRY_SCENARIO=default` means it runs whatever is in the bower.json, which is `"ember": "^2.6.0",`

Seeing as this is between lts-2.4 and 2.8 should we move the line there? Or should we update the bower.json to a different version and remove the default scenario?